controllers/users.js

```diff
@@ -22,14 +22,14 @@ router.get('/:id/spots', function(req, res) {
 // Create a new user
 router.post('/', function(req, res) {
   if (!req.body.hasOwnProperty('email') ||
-      !req.body.hasOwnProperty('password') ||
+      !req.body.hasOwnProperty('uid') ||
         !req.body.hasOwnProperty('firstName') ||
           !req.body.hasOwnProperty('lastName')) {
     res.status(400).json({error: "Missing expected body parameter."});
     return;
   }
 
-  User.create(req.body.email, req.body.password, req.body.firstName,
+  User.create(req.body.email, req.body.uid, req.body.firstName,
```

Comment:
Oh so we want to know email whether or not they are using Facebook auth? You know there might be a way to get email from facebook actually. But ya lets just consider email as some "required user info" and how you to choose to log in is a completely different issue